provider/source/src/currency/extended.rs

```diff
@@ -70,8 +80,16 @@ impl crate::IterableDataProviderCached<CurrencyExtendedDataV1Marker> for SourceD
                 .read_and_parse(&locale, "currencies.json")?;
 
             let currencies = &currencies_resource.main.value.numbers.currencies;
-            for key in currencies.keys() {
-                if let Ok(attributes) = DataMarkerAttributes::try_from_string(key.clone()) {
+            for (currency, patterns) in currencies {
+                if patterns
+                    .display_name
```

Comment:
In this case, count `other` and `displayname` must exist. 

add this comment:
According to [Unicode Technical Standard #35](https://unicode.org/reports/tr35/tr35-numbers.html), the other unit pattern and the displayname must be present in the data.

Reply:
Do you have a more specific link? I can't find where it says that.

Reply:
It doesn't really matter what the standard says, we're skipping them because we're building display names data, and if there is no display name we cannot do that.

Reply:
the problem in the standard that there is no more specific links on it, but no problem for now